File: 168. Excel Sheet Column Title/solution.js
/**
 * @param {number} n
 * @return {string}
 */
var convertToTitle = function(n) {
  let result = "";
  while (n-- > 0) {
    result = String.fromCharCode(65 + (n % 26)) + result;
    n = Math.floor(n / 26);
  }
  return result;
};

File: 168. Excel Sheet Column Title/solution.go
func convertToTitle(n int) string {
	bytes := make([]byte, 0)
	for ; n > 0; n /= 26 {
		if n%26 == 0 {
			bytes = append([]byte{byte('Z')}, bytes...)
			n -= 26
		} else {
			bytes = append([]byte{byte(n%26 + 64)}, bytes...)
			n -= n % 26
		}
	}
	return string(bytes)
}